manim/_config/main_utils.py

```diff
@@ -429,6 +429,10 @@ def _parse_args_no_subcmd(args: list) -> argparse.Namespace:
         help="Frame rate to use when generating keyframe data for animations that use updaters while using the WebGL frontend",
     )
 
+    parser.add_argument(
+        "--debug", action="store_const", help="triggers visual debug mode.", const=True
```

Comment:
Even though this file doesn't exist in the click branch
```suggestion
        "--debug", action="store_true", help="triggers visual debug mode."
```